apps/service/src/rgbpp/types.ts

```diff
@@ -72,12 +71,9 @@ export interface RgbppTransferAllReq {
   };
 }
 
-export interface RgbppTransferAllRes extends Omit<RgbppTransferAllTxsResult, 'transactions'> {
-  transactions: RgbppTransferAllGroupWithStringCkbVtx[];
-}
-
-export interface RgbppTransferAllGroupWithStringCkbVtx extends Omit<RgbppTransferAllTxGroup, 'ckb'> {
-  ckb: Omit<RgbppTransferAllTxGroup['ckb'], 'virtualTxResult'> & {
-    virtualTxResult: string;
-  };
+export interface RgbppTransferAllResp {
+  ckbVirtualTxResult: string;
+  psbtHex: string;
```

Comment:
```suggestion
  btcPsbtHex: string;
```

Reply:
PSBT typically refers to BTC by default, so not adding the BTC prefix wouldn't confuse.

Reply:
That's fine with me, but notice there is a `btcPsbtHex` declared in the types: https://github.com/ckb-cell/rgbpp-sdk/blob/develop/apps/service/src/rgbpp/types.ts#L18-L23